kong/db/schema/entities/schemas.lua

```diff
@@ -0,0 +1,40 @@
+local typedefs = require "kong.db.schema.typedefs"
+local schema = require "kong.schema"
+
+---@class kong.db.schema.entities.schema : table
+---
+---@field id         string
+---@field name       string
+---@field created_at number
+---@field updated_at number
+---@field tags       string[]
+---@field type       kong.schema.strategy.name
+---@field schema     string
+
+
+return {
+  name = "schemas",
```

Comment:
The name `schemas` is generic enough that it _might_ yield confusion and/or naming collisions. Should we consider calling it `user_schemas`?

Reply:
(to clarify, I think the table should remain `schemas` and the schema names themselves should contain namespacing)